packages/kbn-analytics/BUILD.bazel

```diff
@@ -42,7 +42,7 @@ jsts_transpiler(
   name = "target_web",
   srcs = SRCS,
   build_pkg_name = package_name(),
-  config_file = ".babelrc.browser"
+  web = True,
```

Comment:
I guess the default env is `node` that's why you removed `"presets": ["@kbn/babel-preset/node_preset"]`. Didn't you? 
Would you mind providing some details in the PR description, please?

Reply:
So we removed the babel config files and move the little amount of config we had there to inline configuration into that underlying rule by default https://github.com/elastic/kibana/pull/110620/files#diff-7d28a377f4c3860af15c35e48654bfdfb4292ce634b776066e8e7cdb42936645R29

So yeah, web is false by default which means we will be using `@kbn/babel-preset/node_preset` as a preset in that case